wallet/src/repository/error.rs

```diff
@@ -44,6 +44,12 @@ pub enum Error {
     HashParseError(#[cause] types::HashParseError),
     #[fail(display = "failed creating a transaction: {}", _0)]
     TransactionCreation(#[cause] TransactionError),
+    #[fail(display = "Bech32 serialization error: {}", _0)]
+    Bech32(#[cause] bech32::Error),
+    #[fail(display = "Crypto operation failed: {}", _0)]
+    CryptoError(#[cause] crypto::Error),
+    #[fail(display = "Key serialization failed failed")]
```

Comment:
faileddouble :rofl: . And what's the "key" in this context? I guess the master key?

```suggestion
    #[fail(display = "Key serialization failed")]
```

Reply:
good catch. Depends on the database, if the master key exists it refers to the master key. Otherwise, it refers to the account keys, but I will modify the messsage as "master key serialization failed"